eng/code-quality-reports/src/main/java/com/azure/tools/checkstyle/checks/ServiceClientCheck.java

```diff
@@ -261,7 +273,8 @@ private void checkMethodNamingPattern(DetailAST methodDefToken) {
         // prefix of method name that contains all lower letters
         final String prefix = methodName.split("[A-Z]", 2)[0];
         if (!methodName.endsWith("Exists") && !COMMON_NAMING_PREFIX_SET.contains(prefix)) {
-            log(methodDefToken, String.format("Method name ''%s'' should follow a common vocabulary. Refer to Java Spec: %s.",
+            log(methodDefToken, String.format("Method name ''%s'' should follow a common vocabulary. Refer to Java "
+                    + "Spec: %s.",
```

Comment:
might be a good look to have this line break remove, like

"Spec: %s.", methodName, JAVA_SPEC_LINK));